composer.lock

```diff
@@ -12183,5 +12183,5 @@
     "platform-overrides": {
         "php": "7.4"
     },
-    "plugin-api-version": "2.6.0"
+    "plugin-api-version": "2.2.0"
```

Comment:
do you know what is causing this version downgrade?

Reply:
Its related to different composer version on developers environments, its not breaking anything. I need to update composer :)